lib/wanda_web/controllers/executions_controller.ex

```diff
@@ -26,6 +26,6 @@ defmodule WandaWeb.ExecutionsController do
   def list_checks_executions(conn, params) do
     results = Results.list_execution_results(params)
     total_count = Results.count_execution_results(params)
-    json(conn, %{data: results, total_count: total_count})
+    render(conn, data: results, total_count: total_count)
```

Comment:
This should be changed to (at least, the parameters usage):
```
def list_checks_executions(conn, params) do
    executions = Results.list_execution_results(params)
    render(conn, "list_checks_executions.json", executions: executions)
  end
```

Reply:
Looking at the GitHub API they just call an array of results `items`, I'd feel like having a more generic name could be something valuable for users that interact with the API. @fabriziosestito what do you think?

Reply:
`items` is good

Reply:
About the naming, I'm not sure if you could have this:
`render(conn, "list_checks_executions.json", items: items)`
I think it would fail, as phoenix expects here the same name in plural of the view (`ExecutionView` -> `executions`).
`render(conn, "list_checks_executions.json", executions: items)`, would work

Reply:
I was about to rename everything to result, we are using the result context and we have so many different name here. I will add a cleanup PR to show you guys if we merge this.